src/components/MultiselectTags/MultiselectTags.vue

```diff
@@ -0,0 +1,176 @@
+  - along with this program. If not, see <http://www.gnu.org/licenses/>.
+  -
+  -->
+
+<docs>
+## Single tag selector
+
+```vue
+<template>
+	<div class="wrapper">
+		<MultiselectTags v-model="value" :multiple="false" />
+		{{ value }}
+	</div>
+</template>
+
+<script>
+export default {
+	data() {
+		return {
+			value: 1
+		}
+	}
+}
+</script>
+```
+
+## Multiple tag selector
+```vue
+<template>
+	<div class="wrapper">
+		<MultiselectTags v-model="value" :multiple="true" />
+		{{ value }}
+	</div>
+</template>
+
+<script>
+export default {
+	data() {
+		return {
+			value: [1, 2]
+		}
+	}
+}
+</script>
+```
+</docs>
+
+<template>
+	<Multiselect :value="inputValue"
+		:options="tags"
+		:options-limit="5"
+		:placeholder="label"
+		track-by="id"
+		:custom-label="tagLabel"
+		:multiple="multiple"
+		:close-on-select="multiple"
+		:tag-width="60"
+		:disabled="disabled"
+		@input="update">
+		<span slot="noResult">{{ t('core', 'No results') }}</span>
+		<template #option="scope">
+			{{ tagLabel(scope.option) }}
+		</template>
+	</multiselect>
+</template>
+
+<script>
+import { Multiselect } from 'Components/Multiselect'
+import { searchTags } from './api'
+
+export default {
+	name: 'MultiselectTags',
+	components: {
+		Multiselect
+	},
+	props: {
+		label: {
+			type: String,
+			default: t('systemtags', 'Select a tag')
+		},
+		value: {
+			type: [Number, Array],
+			required: true,
+			default: null,
+		},
+		disabled: {
+			type: Boolean,
+			default: false
+		},
+		multiple: {
+			type: Boolean,
+			default: true
+		}
+	},
+	data() {
+		return {
+			tags: []
+		}
+	},
+	computed: {
+		inputValue() {
+			return this.getValueObject()
+		}
+	},
+	async beforeCreate() {
+		try {
+			const result = await searchTags()
+			this.tags = result
+		} catch(error) {
+			console.error.bind(this)
+		}
+	},
+	methods: {
+		getValueObject() {
+			if (this.tags.length === 0) {
+				return []
+			}
+			if (this.multiple) {
+				return this.value.filter((tag) => tag !== '').map(
+					(id) => this.tags.find((tag2) => tag2.id === id)
+				)
+			} else {
+				return this.tags.find((tag) => tag.id === this.value)
+			}
+		},
+		update(value) {
+			if (this.multiple) {
+				/**
+				 * Emitted on input events of the multiselect field
+				 * @type {number|number[]}
+				 */
+				this.$emit('input', value.map((element) => element.id))
+			} else {
+				if (value === null) {
+					this.$emit('input', null)
+				} else {
+					this.$emit('input', value.id)
+				}
+			}
+		},
+		/**
+		 * @param {Object} tag
+		 * @param {string} tag.displayName
+		 * @param {bool} tag.userVisible
+		 * @param {bool} tag.userAssignable
+		 */
+		tagLabel({ displayName, userVisible, userAssignable }) {
+			if (userVisible === false) {
+				return t('systemtags', '%s (invisible)').replace('%s', displayName)
+			}
+			if (userAssignable === false) {
+				return t('systemtags', '%s (restricted)').replace('%s', displayName)
```

Comment:
```suggestion
				return t('systemtags', '{displayName} (restricted)', { displayName })
```

Reply:
I'd say we keep it like this as this is the current way the translation is stored in the systemtags app.

Reply:
Ah, yeah, the good old l10n issue! :see_no_evil: 
Forgot about this! 

Maybe add a TODO?

Reply:
Added